tests/inference/MethodInvocation.java

```diff
@@ -1,13 +1,14 @@
 import universe.qual.*;
 
 class B {
```

Comment:
Please move these test-specific, short classes into the top-level class of the test, to avoid accidental name clashes.